docs/plugins/repository-azure.asciidoc

```diff
@@ -90,6 +90,25 @@ cloud:
 
 In this example, timeout will be 10s for `my_account1` and 30s for `my_account2`.
 
+You can define a proxy using the following settings:
+
+[source,yaml]
+----
+cloud:
+    azure:
+        storage:
+            my_account:
+                account: your_azure_storage_account
+                key: your_azure_storage_key
+                proxy:
+                    host: proxy.host
+                    port: 8888
+                    type: http
+----
+
+Supported values for `proxy.type` are `direct`, `http` or `socks`. Defaults to `direct` (no proxy).
```

Comment:
I don't think we need to document the proxy.type `direct`, because it actually does nothing on Azure client side. We could instead check that proxy.type must be either http or socks and require non null host and port if type is set? Similarly, if host (or port) is set, port (or host) and type are required?